int32 label
---
bool success
string message
